File: Assignment No38/Program5.c
#include<stdio.h>

int Mult(int iNo)
{
    static int iMul = 1;
    int iDigit = 0;

    if(iNo != 0)
    {
        iDigit = iNo % 10;
        
        if(iDigit == 0)
        {
            iDigit = 1;
        }
        
        iMul = iMul * iDigit;
        iNo = iNo / 10;
        Mult(iNo);
    }
    return iMul;
}

int main()
{
    int iValue = 0, iRet = 0;

    printf("Enter number\n");
    scanf("%d",&iValue);

    iRet = Mult(iValue);

    printf("%d",iRet);

    return 0;
}

File: Assignment No31/Program5.java
import java.lang.*;
import java.util.*;

class Bitwise
{
    public int OnBit(int iNo)
    {
        int iMask = 0X0000000F;
        int iResult = 0;

        iResult = iNo | iMask;

        return iResult;
    }
}

class Program4
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number");
        int iValue = sobj.nextInt();

        Bitwise bobj = new Bitwise();
        
        int iRet = bobj.OnBit(iValue);
        System.out.println("Updated number is:"+iRet);

    }
}